### DISCUSSION TITLE
Ai最難取代的10種行業

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#745] [General]
雖然可能沒有很準
但大家還是可以看一下自己有沒有機會在上面
有的話還至少安心一點（？？
不然的話祝大家都可以努力讀到博士當教授～
https://www.gvm.com.tw/article/46062

**Comment by alvin1226** [comment]
很多在上面的都是AI最難學習的情感方面的東西呢，我想這就是無法被取代的原因吧
